packages/keystone/fields/AddressPartWithType/Implementation.js

```diff
@@ -0,0 +1,16 @@
+const { JsonImplementation } = require('../Json/Implementation')
+
+class AddressPartWithTypeImplementation extends JsonImplementation {
```

Comment:
it will be great to move this field inside the plugin. You don't need to use this field without the plugin contex